code/game/objects/items/devices/taperecorder.dm

```diff
@@ -310,6 +310,20 @@
 	canprint = FALSE
 	addtimer(VARSET_CALLBACK(src, canprint, TRUE), 30 SECONDS)
 
+/obj/item/taperecorder/verb/WipeTapeInRecorder()
```

Comment:
instead of having this be a verb, you might be better off having this as a `attack_hand_secondary` proc on the object itself, if it doesn't already have something mapped to that.